pay_android/android/src/main/kotlin/io/flutter/plugins/pay_android/GooglePayHandler.kt

```diff
@@ -183,7 +183,10 @@ class GooglePayHandler(private val activity: Activity) :
                 }
 
                 Activity.RESULT_CANCELED -> {
-                    // The user cancelled the payment attempt
+                    loadPaymentDataResult!!.error(
+                            "paymentCanceled",
+                            "User canceled payment authorisation.",
```

Comment:
It would be great to have the same error message for both platforms
```suggestion
                            "User canceled payment authorization",
```